content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,76 @@
+
+## Telemetry
+
+**Added** support for Envoy gRPC Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als).
+**Added** a Grafana dashboard for Citadel monitoring.
+**Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+**Added** control plane metrics to monitor Istio configuration state.
+**Added** telemetry reporting for traffic destined to `Passthrough` and `BlackHole` cluster.
+**Added** alpha support for in-proxy generation of service metrics using Prometheus.
+**Added** alpha support for environmental metadata in Envoy node metadata.
+**Added** alpha support for Proxy Metadata Exchange.
+**Added** alpha support for OpenCensus trace driver.
+**Improved** the mesh dashboard to provide monitoring of Istio configuration state.
+**Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+**Removed** deprecated Adapter and Template custom resource definitions.
+**Deprecated** Mixer adapters. Extensions are expected to begin transition to new extension model. Legacy Mixer integration support will be maintained for 1.3 and 1.4.
+**Deprecated** Deprecated HTTP API Spec. The support for producing API attributes will be be removed in Istio 1.4.
+
+## Policy
+
+**Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration Management
+
+**Added** the ability to configure Galley to use newer configuration processing.
+**Fixed** an issue in Galley where too many gRPC pings close connections.
+**Improved** Galley to avoid control plane upgrade failures.
+
+
+## istioctl
+
+**Added** many new istioctl experimental subcommands.
+- istioctl kube-uninject -f <descriptor.yaml> The inverse of kube-inject; removes Istio configuration from a Deployment descriptor.
+- istioctl describe pod <pod> Produce description of the Istio configuration affecting a pod
+- istioctl manifest apply Generates and applies Istio install manifest
+- istioctl manifest diff Show the the difference between two Istio manifests
+- istioctl manifest generate Generate Istio install manifest
+- istioctl manifest migrate Convert Helm install values to IstioControlPlane format
+- istioctl manifest versions List supported Istio versions
+- istioctl profile diff Show the difference between two Istio profiles
+- istioctl profile dump Show an Istio configuration profile
+- istioctl profile list Lists Istio configurations
+- istioctl add-to-mesh add local Kubernetes service or remote services(eg: services on VM) into Istio service mesh
+- istioctl remove-from-mesh remove local Kubernetes service or remote services(eg: services on VM) from Istio service
+**Improved** the istioctl experimental subcommands metrics, convert-ingress, and dashboard such that they are now supported Istio features.
+
```

Comment:
## Installation & Upgrades:

** Added **
- [Istio Operator CLI](https://preliminary.istio.io/docs/setup/install/operator/), a convenient new way to install and manage the Istio control plane.